doc/using_mxnet.rst

```diff
@@ -312,6 +312,15 @@ For more information on the runtime environment, including specific package vers
 If there are other packages you want to use with your script, you can include a ``requirements.txt`` file in the same directory as your training script to install other dependencies at runtime.
 Both ``requirements.txt`` and your training script should be put in the same folder.
 You must specify this folder in ``source_dir`` argument when creating an MXNet estimator.
+
+The function of installing packages using ``requirements.txt`` is supported for all MXNet versions during training.
+When serving an MXNet model, support for this function varies with MXNet versions.
+For MXNet 1.6.0 or newer, ``requirements.txt`` must be under folder ``code``.
+The SageMaker MXNet Estimator will automatically save ``code`` in ``model.tar.gz`` after training (assuming you set up your script and ``requirements.txt`` correctly as stipulated in the previous paragraph).
+In the case of bringing your own trained model for deployment, you must save ``requirements.txt`` under folder ``code`` in ``model.tar.gz`` yourself or specify it through ``dependencies``.
+For MXNet 1.4.1, ``requirements.txt`` is not supported for inference.
+For MXNet 0.12.1-1.3.0, ``requirements.txt`` must be in ``source_dir``.
```

Comment:
So... 1.3.0 isn't usually offered, just 1.3.1. So maybe this should be changed.
Also, the versions go back 0.11.0.

Reply:
0.12.1 was the first version SageMaker supported, and 1.3.0 was the 1.3 version that we picked: https://github.com/aws/sagemaker-python-sdk#mxnet-sagemaker-estimators